dotcom-rendering/src/components/YoutubeAtom/YoutubeAtom.test.tsx

```diff
@@ -79,13 +77,13 @@ describe('YoutubeAtom', () => {
 			</ConfigProvider>
 		);
 		const { getByTestId } = render(atom);
-		const overlay = getByTestId('youtube-overlay-ZCvZmYlQD8-xyz');
+		const overlay = getByTestId(/^youtube-overlay-ZCvZmYlQD8-\d+$/);
```

Comment:
Would it be more precise to expect the sequential id that is being set?

```suggestion
		const overlay = getByTestId('youtube-overlay-ZCvZmYlQD8-0');
```

Reply:
I assume the tests are inter-dependent because we now have module level state? So each test sees an ever increasing index because module level state keeps incrementing the index every time we render an atom.

This isn't a problem with the tests for one atom but when there are multiple atoms in a test we just need to be sure the right atom is being selected and asserted against.

I can see the current multiple atom test gets around this by using unique videoids but I imagine a test with duplicate videoids with the current selectors would match two atoms rather than one.

I think its okay for now but it is implicit knowledge required when working with the ids.

Reply:
Addressed in #10221